ts/parts-more/Polar.ts

```diff
@@ -654,6 +657,25 @@ if (seriesTypes.column) {
         if (xAxis.isRadial) {
             points = series.points;
             i = points.length;
+            yMin = yAxis.translate(yAxis.min as any);
+            yMax = yAxis.translate(yAxis.max as any);
+
+            // Finding a correct threshold
+            if (chart.inverted && H.isNumber(threshold)) {
+                (thresholdAngleRad as any) = yAxis.translate(threshold);
```

Comment:
We should avoid to casting `as any`

Reply:
Corrected by adding a simple check if thresholdAngleRad returns a truthy value.